test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -181,8 +204,35 @@ if [ "$FLOW_AGGREGATOR" == "true" ]; then
             ssh -F ssh-config k8s-node-control-plane kubectl apply -f elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
             LOGSTASH_CLUSTER_IP=$(ssh -F ssh-config k8s-node-control-plane kubectl get -n elk-flow-collector svc logstash -o jsonpath='{.spec.clusterIP}')
             ELK_ADDR="${LOGSTASH_CLUSTER_IP}:4739:udp"
-
             $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $ELK_ADDR > "${FLOW_AGG_YML}"
+
+        elif [[ $FLOW_COLLECTOR == "clickhouse" ]]; then
+            echo "Deploy ClickHouse flow collector"
+            # Generate manifest
+            $THIS_DIR/../../../../hack/generate-manifest-flow-visibility.sh --mode dev > "${FLOW_VIS_YML}"
+            $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -ch > "${FLOW_AGG_YML}"
+
+            # Collect images info and push to nodes
```

Comment:
I remember Shawn told me that, if don't push images to nodes in advance, it will cause some delay between apply yml and pod/service getting ready.
@wsquan171

Reply:
I changed it to only push antrea, fa, clickhouse-monitor images to nodes.